main.nf

```diff
@@ -107,9 +107,17 @@ if(params.data_download.run == "True"){
                 --has-ontology                 
         """
     }
+    
+}else{
```

Comment:
Not sure why we need an `else` statement here.

Reply:
I was setting the cross-validation input channels depending on data_download.run = T/F. But maybe it's a better idea to have it depending on corss_validation.run. This looks better right @a-solovyev12 ?

Reply:
I think you don't need it at all as it leads to faulty logic. When you run data import part, you also want to run the rest of the workflow, which won't happen when else{} block is there

Reply:
Wait a second, it's fine, you simply need to remove the `if(params.cross_validation.run == "True")` line. We assume there are only two possibilities here - either data comes from the import step or you provide it yourself.

Reply:
Ok in that case, there's no point in naming parameters `cross_validation.cv_ref_10x_dir`. We can just name it `input_data.ref_10X_dir` or something more general.

Reply:
yes, that's a good idea